### PR TITLE
Swift pm support

### PR DESCRIPTION
**Pull request by nidhik**
## Zube
https://zube.io/muxinc/mux/c/16959

### CODE REVIEW

File: apps/DemoApp/Podfile.lock [hunk 1]
```diff
@@ -11,24 +11,32 @@ PODS:
 DEPENDENCIES:
   - GoogleAds-IMA-iOS-SDK (~> 3.14)
   - Mux-Stats-AVPlayer (from `../../`)
+  - Mux-Stats-Core (from `https://github.com/muxinc/stats-sdk-objc.git`, branch `swift-pm-support`)
```
Comment (nidhik) [inline apps/DemoApp/Podfile.lock:14]: note to self: remove this before release

File: MUXSDKStats/Podfile [hunk 2]
```diff
@@ -6,13 +6,15 @@ target 'MUXSDKStats' do
   # use_frameworks!
 
   # Pods for MUXSDKStats
-  pod 'Mux-Stats-Core', '~>3.1.0'
+  # pod 'Mux-Stats-Core', '~>3.1.0'
+  pod 'Mux-Stats-Core', :git => 'https://github.com/muxinc/stats-sdk-objc.git', :branch => 'swift-pm-support'
```
Comment (nidhik) [inline MUXSDKStats/Podfile:10]: note to self: remove this before release

File: Package.swift [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        .tvOS(.v9)
+    ],
+    products: [
+        .library(name: "MUXSDKStats", targets: ["MUXSDKStatsTargets"])
+    ],
+    dependencies: [
+        .package(
+            name: "MuxCore",
+            url: "https://github.com/muxinc/stats-sdk-objc.git",
+            .branch("swift-pm-support")
```
Comment (dylanjha) [inline Package.swift:18]: note: use `from: 3.2` syntax here instead of branch

File: Mux-Stats-AVPlayer.podspec [hunk 4]
```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.license          = 'Apache 2.0'
   s.author           = { 'Mux' => 'ios-sdk@mux.com' }
 
-  s.dependency 'Mux-Stats-Core', '~>3.1.0'
+  s.dependency 'Mux-Stats-Core', '~>3.2'
```
Comment (dylanjha) [inline Mux-Stats-AVPlayer.podspec:17]: correct semver change here 👍

**Review by dylanjha — APPROVED** [review]

**Pull request merged**